wrappers/vue/test/hotTable.spec.ts

```diff
@@ -206,6 +206,13 @@ describe('Updating the Handsontable settings', () => {
 
     expect(testWrapper.vm.$children[0].hotInstance.getData()).toEqual([[22, 32, 42]]);
     expect(newHotSettings).toBe(null);
+
+    testWrapper.vm.removeRow();
```

Comment:
I don't understand the reliability of this test.
Function `removeRows` corresponds to
line 167 
 ```
 removeRow: function () {
           this.data.pop ()
         }

```
Why do we consider decreasing the dataset as deleting a row?

Reply:
We consider. However, it doesn't work good for nested rows, as I mentioned (bug). This test checks refers to removing the last row.